smartseq2_single_sample/README.md

```diff
@@ -0,0 +1,62 @@
+# SmartSeq2 scRNA-Seq Pipeline
+This pipeline performance the following tasks:
+- Align reads of each sample to reference genome (using STAR)
+- Perform quality control on generated BAM files (using Picard)
+- Count reads in features (using FeatureCounts)
+- Count reads in features (using RSEM)
+
+# Download
+Use git clone `git clone git@github.com:HumanCellAtlas/skylab.git`
+# Requirements
+## Dockers
+- STAR(2_5_3a) docker: `humancellatlas/star_dev:v1`
+- Picard(2.10.10) docker: `humancellatlas/picard`
+- RSEM docker: `humancellatlas/rsem`
+## Genome Reference
+Generated genome index files using instructions, seen in [STAR Manual](https://github.com/alexdobin/STAR/blob/master/doc/STARmanual.pdf) and [Build RSEM index](http://deweylab.biostat.wisc.edu/rsem/rsem-prepare-reference.html). The genome indexes and refernece files saved to google buckets and need only be generated once for each genome/annotaitons combinations. Besides to the genome indexes, you need:
```

Comment:
indexes -> indices
reference has a typo
annotations has a typo and should be singular 'annotation'
combinations can be singular
I think it'd be best to replace `Besides the genome indexes, you need:` with 
`Other inputs include:`